Tests/PerformanceTests.swift

```diff
@@ -70,7 +70,8 @@ final class PerformanceTests: XCTestCase {
         // the layers aren't deallocated until the end of the test run,
         // which causes memory usage to grow unbounded.
         CATransaction.begin()
-        setupAnimationView(with: animation, configuration: .init(renderingEngine: .mainThread))
+        let animationView = setupAnimationView(with: animation, configuration: .init(renderingEngine: .mainThread))
+        animationView.animationLayer!.display()
```

Comment:
Maybe add a comment here and below about how this line is required to trigger the layer to perform its pending setup?